### PR TITLE
paportal rolling workflow improvements

### PR DESCRIPTION
**Pull request by ameyaapte1**
Delete manifest before uploading data to emulate fresh upload of portal data

### CODE REVIEW

File: .github/workflows/paportal-rolling-instance-actions.yml [hunk 1]
```diff
@@ -82,6 +82,10 @@ jobs:
         website-id: f46b70cc-580b-4f1a-87c3-41deb48eb90d
         download-path: ./paportal-download
 
+    - name: Delete portal manifest before uploading
+      working-directory: ./paportal-download
+      run: rm ./starter-portal/.portalconfig/*-manifest.yml
```
Comment (davidjenni) [inline .github/workflows/paportal-rolling-instance-actions.yml:87]: better to define a env variable (GH Flows' concept of 'pipeline' variables) once with the common path to the local starter portal folder, and the have the 2 delete steps and the upload/download steps use the common path from the env var expression

**Review by davidjenni — CHANGES_REQUESTED** [review]

File: .github/workflows/paportal-rolling-instance-actions.yml [hunk 2]
```diff
@@ -109,6 +115,8 @@ jobs:
       WF_USERNAME: davidjen@ppdevtools.onmicrosoft.com
       WF_APPID: 8a7729e0-2b71-4919-a89a-c789d0a9720a
       WF_TENANT_ID: 3041a058-5110-495a-a575-b2a5571d9eac
+      PORTAL_WORKING_DIRECTORY: ./paportal-download
```
Comment (davidjenni) [inline .github/workflows/paportal-rolling-instance-actions.yml:118]: nit: env.Portal_working_directory and ...manifest_file_path

**Review by davidjenni — APPROVED** [review]

File: .github/workflows/paportal-rolling-instance-actions.yml (hunk 2 above)
Comment (ameyaapte1) [reply .github/workflows/paportal-rolling-instance-actions.yml:118]: This job runs on linux, don't we have to reinitialize the variables? Or we can reuse env variables from the previous job?

Comment (davidjenni) [reply .github/workflows/paportal-rolling-instance-actions.yml:118]: well, does your workflow work when you queue a flow for your PR branch?
Also: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env

Comment (ameyaapte1) [reply .github/workflows/paportal-rolling-instance-actions.yml:118]: Yes it is running as expected, here's a run against my branch, https://github.com/microsoft/powerplatform-actions/actions/runs/2230532791

Comment (ameyaapte1) [reply .github/workflows/paportal-rolling-instance-actions.yml:118]: Will take up this in another PR if required

**Pull request merged**